tools/install/DynamoInstaller.iss

```diff
@@ -194,13 +195,23 @@ begin
       result := false;
     end
 
+  // if old EXE version of 0.8.0 is installed, uninstall it
+  sUnInstPath := 'Software\Microsoft\Windows\CurrentVersion\Uninstall\{6B5FA6CA-9D69-46CF-B517-1F90C64F7C0B}_is1'
+  sUnInstallString := ''
+  exeVersion := ''
+  RegQueryStringValue(HKLM, sUnInstPath, 'UnInstallString', sUninstallString)
+  RegQueryStringValue(HKLM, sUnInstPath, 'DisplayVersion', exeVersion)
+  if (sUnInstallString <> '') and (exeVersion = '0.8.0') then
+	Exec(RemoveQuotes(sUnInstallString), '/VERYSILENT /NORESTART /SUPPRESSMSGBOXES /UPDATE', '', SW_HIDE, ewWaitUntilTerminated, iResultCode);
+
+
   sMsg2 := ExpandConstant(' In order to proceed with the installation, you need to uninstall {#ProductName} {#Major}.{#Minor} manually.')	
   sUnInstPath := ExpandConstant('Software\Microsoft\Windows\CurrentVersion\Uninstall\{#ProductName} {#Major}.{#Minor}');
   sUninstallString := '';
-  RegQueryStringValue(HKLM, sUnInstPath, 'UnInstallString', sUninstallString);
+  RegQueryStringValue(HKLM64, sUnInstPath, 'UnInstallString', sUninstallString);
     if (sUninstallString <> '') then
 	begin
-		if not RegQueryDWordValue(HKLM, sUnInstPath, 'RevVersion', revision) then
```

Comment:
We also need to block installation of lower version after higher version is already installed, e.g. 0.8.2 can't be installed when 0.9 is installed.